swarm.cabal

```diff
@@ -79,12 +80,113 @@ common ghc2021-extensions
       -- Not GHC2021, but until we get \cases we use \case a lot
       LambdaCase
 
-library
+
+library boolexpr
```

Comment:
What's our plan for `boolexpr`?  I don't think we can have an internal library called `boolexpr` when there is already a package on Hackage with that name.

Reply:
@kostmo @byorgey let's discuss this in #934 :eyes: